packages/changed-elements-react/src/api/VersionCompareTiles.ts

```diff
@@ -167,112 +167,6 @@ export class Provider
     refreshEvent.raiseEvent();
   }
 
-  /** Variables to keep emphasized elements data */
-  private _eeIsolated: Id64Set | undefined;
-  private _eeHidden: Id64Set | undefined;
-  private _eeEmphasized: Id64Set | undefined;
-  private _lastJson: EmphasizeElementsProps | undefined;
-
-  /** returns copy of previous ee*/
-  public get lastJson(): EmphasizeElementsProps | undefined {
-    if (this._lastJson) {
-      // return a copy of the last json
-      return JSON.parse(JSON.stringify(this._lastJson));
-    }
-    return undefined;
-  }
-
-  /**
-   * Handle merging emphasized elements changes and clear them
-   * This is necessary because non-version compare tools may want to change
-   * visibility using EmphasizeElements class, which will result in clashes of
-   * behavior between the two providers, so we need to merge the operations properly
-   */
-  public handleEmphasizedElements = (viewport: Viewport) => {
```

Comment:
Why are you removing these? These functions are meant to allow using emphasize tools *on top* of what version compare emphasis provides, that's why it maintains its own isolate, hidden and emphasized list. I don't see how that is being replaced by your logic?

Reply:
Tiles is handling more things than it needs to.

"Does it make sense for this to be in VersionCompareTiles? It seems more like something the frontstage manager should be doing."

Tiles was handling this. Did I misunderstand your comment? All I did was add a getter for that already existed in tiles.

Reply:
So, I moved the logic out based on your comment. Also, I saw no references to these member variables and public methods.

Reply:
Should I undo and put back in tiles